test/format/import.c

```diff
@@ -0,0 +1,215 @@
+    create_database(cmd, &import_conn);
+
+    /*
+     * Open two sessions, one for usual test database connection and one for the import database
+     * connection
+     */
+    testutil_check(import_conn->open_session(import_conn, NULL, NULL, &import_session));
+    testutil_check(conn->open_session(conn, NULL, NULL, &session));
+
+    /* Create new table and populate with data in import database connection */
+    testutil_checkfmt(
+      import_session->create(import_session, IMPORT_URI, IMPORT_TABLE_CONFIG), "%s", g.uri);
+    populate_table(import_session);
+
+    /* Grab metadata information for import table from import database connection */
+    get_file_metadata(import_session, &file_config, &table_config);
+
+    while (!g.workers_finished) {
+        period = mmrand(NULL, 1, 10);
+
+        /* Copy table into usual database directory */
+        copy_file_into_directory(session, IMPORT_DIR, "import.wt");
+
+        /* Perform import with either repair or file metadata */
+        memset(buf, 0, sizeof(buf));
+        import_value = mmrand(NULL, 0, 1);
+        if (import_value == 0) {
+            testutil_check(__wt_snprintf(buf, sizeof(buf), "import=(enabled,repair=true)"));
+            if ((ret = session->create(session, IMPORT_URI, buf)) != 0)
+                testutil_die(ret, "session.import", ret);
+        } else {
+            testutil_check(__wt_snprintf(buf, sizeof(buf),
+              "%s,import=(enabled,repair=false,file_metadata=(%s))", table_config, file_config));
+            if ((ret = session->create(session, IMPORT_URI, buf)) != 0)
+                testutil_die(ret, "session.import", ret);
+        }
+
+        verify_import(session);
+
+        /* Drop import table, so we can perform import next iteration */
+        while ((ret = session->drop(session, IMPORT_URI, NULL)) == EBUSY) {
+            __wt_yield();
+        }
+        testutil_check(ret);
+        while (period > 0 && !g.workers_finished) {
+            --period;
+            __wt_sleep(1, 0);
+        }
+    }
+    wts_close(&import_conn, &import_session);
+    testutil_check(session->close(session, NULL));
+    return (WT_THREAD_RET_VALUE);
+}
+
+/*
+ * verify_import --
+ *     Verify all the values inside the imported table.
+ */
+static void
+verify_import(WT_SESSION *session)
+{
+    WT_CURSOR *cursor;
+    WT_DECL_RET;
+    int iteration, key, value;
+
+    iteration = 0;
+    testutil_check(session->open_cursor(session, IMPORT_URI, NULL, NULL, &cursor));
+
+    while ((ret = cursor->next(cursor)) == 0) {
+        error_check(cursor->get_key(cursor, &key));
+        testutil_assert(key == iteration);
+        error_check(cursor->get_value(cursor, &value));
+        testutil_assert(value == iteration);
+        iteration++;
+    }
+    testutil_assert(iteration == IMPORT_ENTRIES);
+    scan_end_check(ret == WT_NOTFOUND);
+    testutil_check(cursor->close(cursor));
+}
+
+/*
+ * populate_table --
+ *     populate import table with simple data.
```

Comment:
Minor clean up: "Populate the import table with..."  (Capitalize the first word of the sentence, add 'the'.)